benefit/data/sheets/CaseField/CaseField.json

```diff
@@ -375,10 +375,12 @@
   {"LiveFrom": "10/06/2019", "CaseTypeID": "Benefit", "ID": "sscsHearingRecordings", "Label": "Hearing recordings", "FieldType": "Collection", "FieldTypeParameter": "sscsHearingRecordingDetails", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "10/06/2019", "CaseTypeID": "Benefit", "ID": "requestableHearingDetails", "Label": "Requestable hearings recordings", "FieldType": "DynamicList", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "10/06/2019", "CaseTypeID": "Benefit", "ID": "requestedHearings", "Label": "Requested hearing recordings", "FieldType": "Collection", "FieldTypeParameter": "hearingRecordingRequest", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "10/06/2019", "CaseTypeID": "Benefit", "ID": "releasedHearings", "Label": "Released hearing recordings", "FieldType": "Collection", "FieldTypeParameter": "hearingRecordingRequest", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "10/06/2019", "CaseTypeID": "Benefit", "ID": "citizenReleasedHearings", "Label": "Released hearing recordings", "FieldType": "Collection", "FieldTypeParameter": "hearingRecordingRequest", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "10/06/2019", "CaseTypeID": "Benefit", "ID": "dwpReleasedHearings", "Label": "Released hearing recordings", "FieldType": "Collection", "FieldTypeParameter": "hearingRecordingRequest", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "requestedHearingsTextList", "Label": "Requested hearings", "FieldType": "Text", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "releasedHearingsTextList", "Label": "Released hearings", "FieldType": "Text", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "09/06/2020", "CaseTypeID": "Benefit", "ID": "hearingRecordingRequestOutstanding", "Label": "Hearing recording request outstanding?", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "requestingParty", "Label": "Requesting party", "FieldType": "DynamicList", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "showRequestingPartyPage", "Label": "Show the requesting party page?", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "showRequestingPartyPage", "Label": "Show the requesting party page?", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "10/06/2019", "CaseTypeID": "Benefit", "ID": "processHearingRecordingRequests", "Label": "Please review the hearing recordings", "FieldType": "Collection", "FieldTypeParameter": "processHearingRecordingRequest", "SecurityClassification": "PUBLIC"}
```

Comment:
Shouldn't the label on this page read "Please review the hearing recordings requests?"

Reply:
It should be an empty label. Femi wanted it removed because the title and the field label have the same text and shows up 3 times at the top of the page.
The field label shows up twice since it is a collection, it will show above the disabled `Add new` button